src/common-util/src/main/java/ee/ria/xroad/common/certificateprofile/impl/DnFieldDescriptionImpl.java

```diff
@@ -30,7 +30,8 @@
 import lombok.experimental.Accessors;
 
 /**
- * Default implementation of DnFieldDescription.
+ * Old implementation of DnFieldDescription, that uses inline labels instead
```

Comment:
I think deprecation is too strong, since this still needs to be used if someone wants to create a `DnFieldDescription` for a field that does not have localization key in `DnFieldLabelLocalizationKey`.

I plan to add
```
 * New implementations should prefer {@link LocalizedFieldDescriptionImpl} if
 * needed {@link DnFieldLabelLocalizationKey} exists.
```
to the apidoc instead?

Reply:
Javadoc updated